### PR TITLE
Manubot sections

### PR DESCRIPTION
**Pull request by dhimmel**
I'm off for the weekend!

### CODE REVIEW

**Comment by agitter** [comment]
I'll review this weekend

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Should we make our sub-section titles more descriptive?  Maybe `Writing manuscripts with Manubot`?

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: `blockquotes` is repeated.  We could skip some of the formatting options that are less relevant for manuscripts like `inline code` and `codeblocks`.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: `to a standard`

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: We don't have much space, but should we briefly explain what CSL controls for someone who is not familiar?  We could provide an example of determining how many authors to show, how to format names, etc.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: The next few sentences feel heavy on jargon.  I don't have a good idea for avoiding it though.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: The citation processing is a huge feature.  Should we emphasize it more?  Maybe include a figure where we show how a DOI reference expands to the citation automatically?  Or include an example citation in the text here?

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: I had been using URL citations instead of links.  We should be consistent.  Links might be best.

**Review by agitter — CHANGES_REQUESTED** [review]
I like this.  It's compact but does a good job explaining the approach, which is just what we need.  Most of my comments are discussion points, not strong recommendations.

I'd like to add a sentence or two about how Manubot + continuous analysis are used to check references, build the manuscript, and share HTML and PDF versions.  That was important for deep review.

I also wonder whether this sub-section might fit better after `Contribution workflow`.

Should we discuss other work related to Manubot in the `Discussion`?  That's where I currently have a placeholder.

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md]: I think it makes most sense to hyperlink to services or entities. Citations make sense for "citeable work"... things that generally have at least 3 of the following 4 pieces of metadata: authors, title, publisher, or date.

If a service also has a corresponding citable work, such as Greycite, then you can link and cite.

Along these lines, I'm in favor of just hyperlinking to GitHub repos. When referring to Manubot manuscripts (hosted via GH Pages) a citation is appropriate.

@agitter do you agree?

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md]: Yes we should demonstrate the capability. I will think of whether a figure or textual example makes the most sense. Currently, manubot replaces citations in code-formatted text (which is wrong, but hard to fix).

File: content/02.main-text.md
Comment (agitter) [reply content/02.main-text.md]: @dhimmel Yes, I agree with that.  I created #11 to remind me to convert some of my existing URL citations.

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md]: > We could skip some of the formatting options that are less relevant for manuscripts like inline code and codeblocks.

I think all of these formatting options are relevant and important to scholarly writing. They are just traditionally absent from manuscripts because Word does a poor job supporting them.

Comment (agitter) [reply content/02.main-text.md]: Point taken

**Comment by agitter** [comment]
@dhimmel is this ready for re-review?

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md]: Yes. Once I complete the writing, we should think about restructuring paragraphs and headings.

**Comment by dhimmel** [comment]
> @dhimmel is this ready for re-review?

Yes. I do plan on adding a few more paragraphs, but you should still be able to review the current state.

**Comment by agitter** [comment]
> Yes. I do plan on adding a few more paragraphs, but you should still be able to review the current state.

We don't have much more space in this format.  The [guidelines](https://github.com/greenelab/meta-review/blob/master/GUIDELINES.md) say

> There are no strict limits on the length, but authors are advised to stay below 2000 words

The word count of 2390 in the build is not accurate, but it suggests we're nearly maxed out.  Trimming TODOs will help with a better estimate.

**Comment by dhimmel** [comment]
> There are no strict limits on the length, but authors are advised to stay below 2000 words, two display items (figures, tables etc) and 20 references if possible.

In general, I think eLife is very reasonable. So if we think that the content would be best communicated in over 2000 words, I think we should do that. My inclination would be to write the article that we want to write. And if eLife judges it to be unnecessarily verbose, we can cut it down.

**Comment by agitter** [comment]
Okay, I won't worry about length in this first draft.

We should be careful about making it broadly accessible.  All of us are familiar with the jargon used here so we may want to have someone with a different background read the complete draft.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Introduce Pandoc?  Something like `Manubot relies on Pandoc, software for converting document formats, to extend markdown and enable`

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: ` Manubot's` or `the Manubot` sounds better to me

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Move link to the first mention of Pandoc.

Can this be simplified to `to HTML, PDF, and optionally DOCX output`?

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: This line and the next could be removed if you simplify the first sentence in the paragraph.

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Should we define `platform`?

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Is `generates updated HTML and PDF documents` more understandable than `rebuilds the manuscript`?

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md:57]: The branches parenthetical is getting fairly technical

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: `timestamp`

File: content/02.main-text.md
Comment (agitter) [inline content/02.main-text.md]: Is that necessarily a good thing?  An [eLife article](https://doi.org/10.7554/eLife.28699) recently pushed back against scientific precedence.

**Review by agitter — COMMENTED** [review]
More intermediate comments.  I'll give a final review when you are done making your planned changes.

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md]: The implications are probably most important legally, when precedence can effect the outcome of an intellectual property dispute or some other question. Personally when evaluating science, I care for quality over novelty. For example, I may cite a more thorough and accessible study over an earlier study. In short, being able to establish precedence is potentially valuable for authors. We're not taking a position of the use of precedence or novelty when judging scientific output. The extent novelty should be used for evaluation (or in legal cases) is a separate question that we don't need to answer.

File: content/02.main-text.md
Comment (dhimmel) [reply content/02.main-text.md]: Will consider. Technically PDF is converted from the HTML using WeasyPrint. But that may be an irrelevant detail.

**Comment by dhimmel** [comment]
Okay this section now contains the main points I want to make about the Manubot. We all probably agree that the various manuscripts sections could be better integrated. Perhaps the right approach would be to merge this and #12 once the in-section issues are fixed. Then we can do more transpositions and section labeling.

**Review by agitter — APPROVED** [review]

**Comment by agitter** [comment]
Looks good to me.  After we merge #12 and reorganize, I'd like to have a third party review the draft.  I'm wondering whether parts of the Manubot section will be understandable to an undergrad.

**Pull request merged**
